src/fomo.jl

```diff
@@ -113,25 +223,50 @@ mutable struct FomoSolver{T, V} <: AbstractFirstOrderSolver
   p::V
   o::V
   α::T
+  params::FOMOParameterSet{T}
 end
 
-function FomoSolver(nlp::AbstractNLPModel{T, V}; M::Int = 1) where {T, V}
+function FomoSolver(nlp::AbstractNLPModel{T, V}; M::Int = FOMO_M, kwargs...) where {T, V}
+  params = FOMOParameterSet{T}(; M = M, kwargs...)
   x = similar(nlp.meta.x0)
   g = similar(nlp.meta.x0)
   c = similar(nlp.meta.x0)
   m = fill!(similar(nlp.meta.x0), 0)
   d = fill!(similar(nlp.meta.x0), 0)
   p = similar(nlp.meta.x0)
   o = fill!(Vector{T}(undef, M), -Inf)
-  return FomoSolver{T, V}(x, g, c, m, d, p, o, T(0))
+  return FomoSolver{T, V}(x, g, c, m, d, p, o, T(0), params)
 end
 
 @doc (@doc FomoSolver) function fomo(
   nlp::AbstractNLPModel{T, V};
-  M::Int = 1,
+  η1::T = eps(T)^(1 // 4),
```

Comment:
Could it be something like 
```
eval(Meta.parse("$(FOMO_η1))")
```
?

Reply:
That was my first idea, but for some scoping reason it doesn't work :(